COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
```

Comment:
I'm not sure we need a separate file for this. I think that some of the content here should go into README.md (if this is for the community at large) and some parts into CONTRIBUTING.md (e.g., a brief description of fairlearn-proposals; maybe other items too).

Reply:
I thought about this for a while, and I don't have a strong opinion either way. I'm happy to move it into the existing files.